Makefile

```diff
@@ -45,7 +45,8 @@ push-image:
 	docker push $(IMAGE_NAME):$(VERSION)
 
 test: codegen download-test-deps
-	KUBEBUILDER_ASSETS="$(shell setup-envtest --use-env use -p path)" go test -v ./test/...
+	KUBEBUILDER_ASSETS="$(shell setup-envtest --use-env use -p path)" \
+		ginkgo -nodes=4 --randomizeAllSpecs ./test/...
```

Comment:
Is this now also a dev machine dependency?  Should it be added to README/CONTRIBUTING?

Reply:
`ginkgo` is mentioned as a prerequisite in `docs/build.md` already (I didn't remove it when I changed the Makefile [the last time](https://github.com/pulumi/pulumi-kubernetes-operator/pull/319)).